cves/CVE-2015-8218.yml

```diff
@@ -342,7 +365,15 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This vulnerability stemmed from, or at the very least was discovered as a
+    result of refactor. The vulnerable function did not exist at all prior to the
+    VCC, but the code did - it was just duplicated within another function. The
+    refactor pulled that function out and made it it's own function, but missed
+    an error case that was likely handled correctly previously. It is very likely
+    that this was simply missed in code review. More rigorous or specific rules
+    regarding refactoring commits may have prevented this vulnerability from
+    ocurring.
```

Comment:
What do you think about a more robust testing suite as well? Seems to me that, especially considering this edge case had already been recognized and fixed in the past, there should be a test or two to handle the case.

Reply:
I couldn't find any testing for this particular file, so I would think that more testing might have done wonders in general. As a general rule I definitely think that more testing is good